packages/kmod-6.12-nvidia-r570/Cargo.toml

```diff
@@ -0,0 +1,45 @@
+[package]
+name = "kmod-6_12-nvidia-r570"
+version = "0.1.0"
+edition = "2021"
+publish = false
+build = "../build.rs"
+
+[lib]
+path = "../packages.rs"
+
+[package.metadata.build-package]
+package-name = "kmod-6.12-nvidia-r570"
+releases-url = "https://docs.nvidia.com/datacenter/tesla/"
+
+[[package.metadata.build-package.external-files]]
+url = "https://s3.amazonaws.com/EULA/NVidiaEULAforAWS.pdf"
+sha512 = "e1926fe99afc3ab5b2f2744fcd53b4046465aefb2793e2e06c4a19455a3fde895e00af1415ff1a5804c32e6a2ed0657e475de63da6c23a0e9c59feeef52f3f58"
+
+[[package.metadata.build-package.external-files]]
+url = "https://us.download.nvidia.com/tesla/570.133.20/NVIDIA-Linux-x86_64-570.133.20.run"
+sha512 = "c6f443791b14e71273929e6e97660cceeb46e9fbd10a3df624236a2433a99b0c905f89ada7c490f2a6009ff37a715c8585a4e82df75adf0afed5236da39ae5b6"
+force-upstream = true
+
+[[package.metadata.build-package.external-files]]
+url = "https://us.download.nvidia.com/tesla/570.133.20/NVIDIA-Linux-aarch64-570.133.20.run"
+sha512 = "a8e338f551e0086e0de899ccf7f7d8b440eabd8842f347ff778170d54f926a10346d79cedd562fa826cb515def92eef60d84064d41571fefe7eb00fc8d2e727b"
+force-upstream = true
+
+[[package.metadata.build-package.external-files]]
+url = "https://developer.download.nvidia.com/compute/cuda/repos/rhel9/x86_64/nvidia-fabric-manager-570.133.20-1.x86_64.rpm"
+sha512 = "4fda6996671f7dc0264dcd4228510366a32eb5a7be1bdc99c248bb3bec41637e1ec41d6c90fea2b52a28ee92b56dc0a0b6e1d81c63731611cff1392fff058737"
+force-upstream = true
+
+[[package.metadata.build-package.external-files]]
+url = "https://developer.download.nvidia.com/compute/cuda/repos/rhel9/sbsa/nvidia-fabric-manager-570.133.20-1.aarch64.rpm"
+sha512 = "e4b6de518ee47bb86c7dcdedc90d2147c9c55605b41dc6782e6e2e7976ea7b06c60dc881b5ef8eef272e021f54353bacb4ef92f488715205742d62f5c64306da"
+force-upstream = true
+
+[[package.metadata.build-package.external-files]]
+url = "https://raw.githubusercontent.com/NVIDIA/open-gpu-kernel-modules/main/COPYING"
```

Comment:
```suggestion
url = "https://raw.githubusercontent.com/NVIDIA/open-gpu-kernel-modules/570/COPYING"
```

Reply:
Will update, that branch hadn't been cut yet when I started.